packages/cxl-ui/scss/global/cxl-details.scss

```diff
@@ -0,0 +1,3 @@
+cxl-details {
+    display: block;
+}
```

Comment:
But `vaadin/details` is already `block`?

https://github.com/vaadin/web-components/blob/main/packages/details/src/vaadin-details.js

Can drop these global style files? Not sure why they were created in the first place.